src/app.ts

```diff
@@ -0,0 +1,12 @@
+import express from 'express';
+
+const app = express();
+const port = 3000;
+
+app.get('/', (req, res) => {
+  res.send('Hello World!');
+});
+
+app.listen(port, () => {
+  return console.log(`Express is listening at http://localhost:${port}`);
```

Comment:
why using return, can't we directly console log

Reply:
Fixed an issue.